chunky/src/java/se/llbit/chunky/renderer/scene/PathTracer.java

```diff
@@ -156,7 +156,7 @@ public static boolean pathTrace(Scene scene, Ray ray, WorkerState state, int add
 
             Vector4 indirectEmitterColor = new Vector4(0, 0, 0, 0);
 
-            if (scene.emittersEnabled && currentMat.emittance > Ray.EPSILON) {
+            if (scene.emittersEnabled && (!scene.isPreventNormalEmitterWithSampling() || scene.getEmitterSamplingStrategy() == EmitterSamplingStrategy.NONE) && currentMat.emittance > Ray.EPSILON) {
```

Comment:
should this be `&& !(scene.isPreventNormalEmitterWithSampling() || scene.getEmitterSamplingStrategy() == EmitterSamplingStrategy.NONE)`?

Reply:
No, I think this is right. The condition for doing normal emitter (aka not sampled) should be true when we do not prevent ir or when there is no strategy